### ISSUE TITLE
Issue when clicking on a multi-instance button in the installation map

### ISSUE THREAD

**Issue by adamabeshouse** [cBioPortal/cbioportal#8023]
Before:
![image](https://user-images.githubusercontent.com/636232/97882261-69a43f80-1cf1-11eb-8369-408d5ffff6b3.png)

Then I click on one of the multi-instance buttons, and other things disappear
![image](https://user-images.githubusercontent.com/636232/97882302-7a54b580-1cf1-11eb-95f1-b1f9cc52dd39.png)

**Comment by jtquach1** [comment]
![click_mult_marker](https://user-images.githubusercontent.com/33106214/97882693-e59e8780-1cf1-11eb-9513-80fe247302ce.gif)
![click_mult_marker_2](https://user-images.githubusercontent.com/33106214/97882888-27c7c900-1cf2-11eb-9263-c0dcf05119bc.gif)

How is your browser window? I found when reducing the browser window size, I also got this behavior. My offset width and height in the first gif is 1903x953. In the second gif it's 1055x821.
